pymc/sampling/external/__init__.py

```diff
@@ -0,0 +1,16 @@
+#   Copyright 2025 - present The PyMC Developers
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+from pymc.sampling.external.base import ExternalSampler
+from pymc.sampling.external.jax import Blackjax, Numpyro
+from pymc.sampling.external.nutpie import Nutpie
```

Comment:
```suggestion
from pymc.sampling.external.nutpie import Nutpie

__all__ = ["Blackjax", "Numpyro", "Nutpie"]
```

You probably don't need to export the ABC right?

Reply:
not importing it anymore